app/src/App.tsx

```diff
@@ -164,13 +165,20 @@ const Root = (props) => {
   ];
 };
 
+const cortexClient = cortex.createClient({
+  serverBaseUrl: '/cortex',
```

Comment:
This should come from ep.config.json. (cortexApi.path). Although perhaps it's worth removing it from the config if it will never change too.